bindings/nodejs/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# https://www.npmjs.com/package/cmake-js#general
+if (CMAKE_JS_SRC)
+    add_library(cmake_js STATIC ${CMAKE_JS_SRC})
+else()
+    add_library(cmake_js INTERFACE)
+endif()
+target_include_directories(cmake_js INTERFACE SYSTEM ${CMAKE_JS_INC})
+target_link_libraries(cmake_js ${CMAKE_JS_LIB})
+
+# https://www.npmjs.com/package/cmake-js#n-api-and-node-addon-api
+add_library(node_addon_api INTERFACE)
+if (NOT NODE_RUNTIME)
+    find_program(NODE_RUNTIME NAMES node HINTS /usr)
+endif()
+execute_process(COMMAND ${NODE_RUNTIME} -p "require('node-addon-api').include"
+        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+        OUTPUT_VARIABLE NODE_ADDON_API_DIR
+        )
+if (NOT NODE_ADDON_API_DIR)
+    message(FATAL_ERROR "Failed to find node-addon-api headers with node command \"${NODE_RUNTIME}\"")
+endif()
+string(REPLACE "\n" "" NODE_ADDON_API_DIR "${NODE_ADDON_API_DIR}")
+string(REPLACE "\"" "" NODE_ADDON_API_DIR "${NODE_ADDON_API_DIR}")
+target_include_directories(node_addon_api INTERFACE SYSTEM ${NODE_ADDON_API_DIR})
+target_link_libraries(node_addon_api INTERFACE cmake_js)
+
+# Supports Node version >= 12.11.0 https://nodejs.org/api/n-api.html#n_api_n_api_version_matrix
+target_compile_definitions(node_addon_api INTERFACE NAPI_VERSION=5)
+
+if (ENABLE_CLIENT)
```

Comment:
I just skipped the whole file, instead of failing in [4f074](https://github.com/amzn/hawktracer/pull/67/commits/4f07400dc79d7288fe687268ec6b6c2b87908d85).